frontend/src/exams/add-exam-form-controller.vue

```diff
@@ -63,11 +77,14 @@
       DropdownQuestion,
       ExamReceivedQuestion,
       InputQuestion,
+      TimeQuestion,
       NotesQuestion,
-      SelectQuestion
+      SelectQuestion,
+      SelectOffice
     },
     mounted() {
       this.getExamTypes()
+      this.getOffices()
```

Comment:
I added (now, in response to your comment -- I didn't know that endpoint would be restricted) this check to the action itself in the store so its covered irrespective of what component it might be called from.

Reply:
check store.actions.getOffices() the logic is on the action itself